spec/fixtures/vcr/Requesting_support/when_recaptcha_is_invalid/and_the_form_is_invalid/does_not_redirect_to_invalid_recaptcha_path.yml

```diff
@@ -200,4 +200,54 @@ http_interactions:
       encoding: UTF-8
       string: '{"id":"bf70e7b4c5eb436bafd25ae462de41af"}'
   recorded_at: Fri, 24 Mar 2023 13:00:47 GMT
+- request:
```

Comment:
?

Reply:
It's automatically generated by VCR, I can only assume either the old request tape has expired or sth else happened causing this extra request to be recorded.

Reply:
Looks like it's being generated by `spec/system/get_help_spec.rb`.

Don't think a new PR is required but it might be nice to put it in another commit for future devs.

Reply:
Actually, it might be generated because I have Sentry keys added in my .env file (which I needed for some previous work). After removing the env keys, the change is not showing up. I'll remove this from this PR